todo_app/data/session_items.py

```diff
@@ -1,19 +1,28 @@
-import requests
 import os
 import datetime as dt
+import pymongo
 from dotenv import load_dotenv
-from todo_app.classes import Item
+from todo_app.classes import mongoItem
 
-# Load .env variables
-load_dotenv()
-api_key = os.getenv('API_KEY')
-token = os.getenv('TOKEN')
-id_board = os.getenv('ID_BOARD')
+# Secret key loading:
+secret_key = os.getenv('SECRET_KEY')
+
+# Datetime format:
 dtformat = '%Y-%m-%dT%H:%M:%S.%fZ'
 
-# Define base parameters
-base_url = 'https://trello.com/1/'
-payload ={ 'key' : api_key , 'token' : token }
+# Function to connect to mongo
+def connect_mongo():
+    load_dotenv()
+    # MongoDB secrets:
+    mongo_usr = os.getenv('MONGO_USR')
+    mongo_psw = os.getenv('MONGO_PSW')
+    mongo_url = os.getenv('MONGO_URL')
+    default_database = os.getenv('DEFAULT_DATABASE')
```

Comment:
Minor but worth noting this is completely ignored, as you just access the "board" database a few lines down anyway with `db = client.board`
It's worth noting that because of that, you're actually not using the test board in your selenium test either!
You might want to switch that line for 
`db = client[default_database]`